lua/dap/session.lua

```diff
@@ -847,8 +847,9 @@ function Session:initialize(config, adapter)
     end)
   end)
   local sec_to_ms = 1000
+  local sec_to_wait = config.sec_to_wait or 4
```

Comment:
`config` is reserved for debug adapter configuration. Client configuration should either go into the `defaults` table or in this case I think I'd prefer being it part of the `adapter` definition.  For example `adapter.options.initialize_timeout_sec`

Reply:
Sure, I changed this to then be part of the adapter options.